app/assets/javascripts/countries/services/PlaceService.js

```diff
@@ -76,7 +76,7 @@ define([
   // }
 
   var urlDefaultsParams = {
-    options: 'ICB7DQogICJncmlkU3RhdHVzIiA6IHsNCiAgICAgInZpZXciOiAibmF0aW9uYWwiLA0KICAgICAidHJlc2hvbGQiOiAzMCwNCiAgICAgIndpZGdldHMiOnsNCiAgICAgIjEiOiB7DQogICAgICAgImF2ZXJhZ2UiOiBudWxsLA0KICAgICAgICJpbmRpY2F0b3IiOiAxLA0KICAgICAgICJ0cmVzaG9sZCI6IDMwLA0KICAgICAgICJ1bml0IjogbnVsbA0KICAgICB9LA0KICAgICAiMiI6ew0KICAgICAgImF2ZXJhZ2UiOiBudWxsLA0KICAgICAgImluZGljYXRvciI6IDEsDQogICAgICAidHJlc2hvbGQiOjMwLA0KICAgICAgInVuaXQiOiBudWxsDQogICAgIH0NCiAgIH0NCiB9DQp9'
```

Comment:
Can we change this for the clear options and then convert them to B64 instead of saving the result in the code?